ts/packages/ts/src/commands/UpsertCommand.ts

```
@@ -5,65 +5,96 @@ import { ConfigError } from '../utils';
+    const { store, set, schema } = this.parameters;
 
-    await store.init();
+    const errorScope: [string, string][] = [
+      ['UpsertCommand', `store:${store.type};set:${set.path};schema:${schema.name}`],
+      ['parameters.configs', `key:${key};value:${value}`],
+    ];
 
-    const upsertConfigs = _(configs)
-      .entries()
-      .map<Config>(([key, value]) => {
-        const errorScope: [string, string][] = [
-          ['UpsertCommand', `store:${store.type};set:${set.path};schema:${schema.name}`],
-          ['parameters.configs', `key:${key};value:${value}`],
-        ];
+    const cfgu = schema.contents[key];
 
-        const cfgu = schema.contents[key];
+    if (!isFromPipe) {
+      if (!cfgu) {
+        throw new ConfigError(
+          'invalid config key',
+          `key "${key}" must be declared on schema ${schema.name}`,
+          errorScope,
+        );
+      }
+      if (value && cfgu?.template) {
+        throw new ConfigError('invalid config value', `keys declared with template mustn't have a value`, errorScope);
+      }
+    }
 
-        if (!cfgu) {
-          throw new ConfigError(
-            'invalid config key',
-            `key "${key}" must be declared on schema ${schema.name}`,
-            errorScope,
-          );
-        }
+    if (value && cfgu) {
+      try {
+        ConfigSchema.CFGU.VALIDATORS.valueOptions(cfgu, value);
+        ConfigSchema.CFGU.VALIDATORS.valueType(cfgu, value);
+      } catch (error) {
+        throw error?.appendScope?.(errorScope) ?? error;
+      }
+    }
+  }
+
+  async run() {
+    const { store, set, schema, configs, pipe } = this.parameters;
 
-        if (value) {
-          if (cfgu.template) {
-            throw new ConfigError(
-              'invalid config value',
-              `keys declared with template mustn't have a value`,
-              errorScope,
-            );
-          }
+    if (_.isEmpty(configs) && _.isEmpty(pipe)) {
+      return;
```

Why would silence the Error in that case?

---

Works the same like providing an empty config object, does nothing.